storages/storages.go

```diff
@@ -177,3 +171,21 @@ func InitStorages(transport http.RoundTripper, clustersConf config.ClustersMap,
 		respHandler:  respHandler,
 	}, nil
 }
+
+func decorateBackend(transport http.RoundTripper, name string, backendConf config.Backend) (http.RoundTripper, error) {
+	backend := &Backend{
+		transport,
+		*backendConf.Endpoint.URL,
+		name,
+		backendConf.Maintenance,
+	}
+	decoratorFactory, ok := auth.Decorators[backendConf.Type]
+	if !ok {
+		return nil, fmt.Errorf("initialization of backend %s has resulted with error: no decorator defined for type %s", name, backendConf.Type)
```

Comment:
why "has"?

Reply:
Done and redirecting PR to https://github.com/allegro/akubra/pull/67